chroma_core/lib/service_config.py

```diff
@@ -293,7 +293,7 @@ def _setup_rabbitmq_credentials(self):
         self.try_shell(sudo + ["rabbitmqctl", "set_user_tags", settings.AMQP_BROKER_USER, "management"])
 
     def _setup_influxdb(self):
-        influx_service = ServiceControlEL7("influxdb")
+        influx_service = ServiceControl("influxdb")
```

Comment:
Should we also restart influx when restarting chroma? I was looking at postgres and I see the following:

```
 def _restart_pgsql(self):
        postgresql_service = ServiceControl.create("postgresql")
        if postgresql_service.running:
            postgresql_service.reload()
        else:
            postgresql_service.start()
        postgresql_service.enable()
```

Reply:
Influx does restart during setup

Reply:
ok guess I missed that.